test/parallel/test-permission-inspector-brk.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fixtures = require('../common/fixtures');
+const file = fixtures.path('permission', 'inspector-brk.js');
+
+common.skipIfWorker();
+common.skipIfInspectorDisabled();
+
+const { status, stderr } = spawnSync(
+  process.execPath,
+  [
+    '--experimental-permission',
+    '--allow-fs-read=*',
+    '--inspect-brk',
+    file,
```

Comment:
This way there's no need to add another file (test/fixtures/permission/inspector-brk.js)

```suggestion
   '--eval',
   'console.log("Hi")',
```

Reply:
It seems the process does not crash when using `--eval` because the crash is caused by [this](https://github.com/nodejs/node/blob/main/lib/internal/modules/cjs/loader.js#L1425) code.

Reply:
Done !